docs/samples/client/csharp/SampleService/SampleClient/src/Generated/SampleTypeSpecClient.cs

```diff
@@ -72,24 +72,11 @@ public SampleTypeSpecClient(Uri endpoint, ApiKeyCredential keyCredential, Sample
         /// <returns> The response returned from the service. </returns>
         public virtual ClientResult SayHi(string headParameter, string queryParameter, string optionalQuery, RequestOptions options)
         {
-            try
```

Comment:
This should not have changed after you revert the global.json changes try regenerating this again to see if it reverts back